packages/js/product-editor/src/components/section-header/types.ts

```diff
@@ -2,5 +2,4 @@ export interface SectionHeaderProps {
 	description?: string | undefined;
 	sectionTagName: string;
 	title: string;
-	type: 'section' | 'subsection';
```

Comment:
This was only use for the slot names and I removed this ( see above comment for reasoning ).